src/graph-helpers/relationship.js

```diff
@@ -0,0 +1,39 @@
+import relationship from './raw-relationship';
+import { parseArgs } from './raw-relationship';
+import fields from './fields';
+import join from './join';
+
+function wrapCallback(callbackFunction) {
+  return function (schema) {
+    if (schema.name.match(/Connection$/)) {
+      const pageInfo = relationship(schema, 'pageInfo', pageInfoSchema => {
+        return fields(pageInfoSchema);
+      });
+      const edge = relationship(schema, 'edges', edgeSchema => {
+        const edgeFields = fields(edgeSchema);
+        const node = relationship(edgeSchema, 'node', nodeSchema => {
+          return callbackFunction(nodeSchema);
+        });
+
+        return join(edgeFields, node);
```

Comment:
why not `[edgeFields, node].join(', ')`?
saves the need of having to create a new module.